### PR TITLE
Format response data

### PR DESCRIPTION
**Pull request by thescriptninja**
This is to format the response data using marshmallow.

### CODE REVIEW

**Comment by thescriptninja** [comment]
The response looks something like this
Screenshot from 2020-05-24 22-29-24

**Comment by thescriptninja** [comment]
A sample response object is:

```
{
    "link": "https://youtu.be/sh55BDdjxu4",
    "postdata": {
      "share_date": "2018-12-15T01:37:42",
      "likes_count": 6,
      "caption": "Beautiful !"
    },
    "metadata": {
      "song": {
        "image": "https://i.scdn.co/image/ab67616d0000b273a5d0a0cdb04d681040c66ff8",
        "name": "Khayal"
      },
      "artists": [
        {
          "image": "https://i.scdn.co/image/f55c26ec714046d3375dafc2901807533f41f366",
          "name": "Shamoon Ismail"
        }
      ],
      "genre": [
        "indian indie",
        "pakistani indie",
        "pakistani pop",
        "sufi"
      ]
}
```

An array of such objects is returned

File: c3po/api/dto.py [hunk 1]
```diff
@@ -1,11 +1,28 @@
...
+
+class SongDto(Schema):
+    name = fields.String()
+    image = fields.String()
+
+
+post_dto = PostDto()
+artist_dto = ArtistDto()
+song_dto = SongDto()
+genre_dto = GenreDto()
```
Comment (ghostwriternr) [inline c3po/api/dto.py:28]: Where is `GenreDto()`?

Comment (thescriptninja) [reply c3po/api/dto.py:28]: I'll remove it. I had created it earlier but then realized it was unnecessary :sweat_smile:

**Comment by mukul-mehta** [comment]
Everything seems fine. Do update `requirements/common.txt` with `marshmallow`.

**Pull request merged**